extensions/2.0/Vendor/BENTLEY_materials_point_style/schema/material.BENTLEY_material_point_style.schema.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "BENTLEY_material_point_style extension",
+  "type": "object",
+  "description": "Specifies point width for CAD-style visualization.",
+  "allOf": [
+    { "$ref": "glTFProperty.schema.json" }
+  ],
+  "properties": {
+    "width": {
+      "type": "number",
+      "description": "Width of the point in screen pixels. Must be > 0.",
+      "minimum": 0
```

```suggestion
      "type": "integer",
      "description": "Width of the point in screen pixels. Must be > 0.",
      "minimum": 1
```

With has to be a whole number. The JSON schema has an `integer` type.

Description says must be > 0, but minimum says must be >= 0.